spec/unit/bootstrap/definitions_spec.rb

```diff
@@ -0,0 +1,29 @@
+describe FHIR::Definitions do
+  context '.load_extensions'
+
+  context '.get_display' do
+    let(:code) { '306005' }
+    let(:uri) { 'http://snomed.info/sct' }
```

Comment:
I pulled these values from the data that loaded, and tried to find matches. I think we've got a good start by hitting the logic paths that appear to be functional. Would love to expand the tests if there's more that we should cover.